daemon/daemon.py

```diff
@@ -268,4 +281,17 @@ async def persistent_sessions(app):
 
   server.on_startup.append(start_background_tasks)
   server.on_cleanup.append(cleanup_background_tasks)
-  web.run_app(server, host='127.0.0.1', port=args.port)
+
```

Comment:
To test how the errors are reported and shown to users, add here:
`exit(100)` general problem, `exit(101)` aiohttp import error, `exit(102)` certifi import error, or `exit(113)` port blocked error (some users faced this with Commodo antivirus).